docs/rose-suite-run-proposal/cylc-flow-rc.md

```diff
@@ -0,0 +1,176 @@
+# Proposal example of a new style `cylc-flow.rc` file
+
+## References
+[cylc-admin PR #40](https://github.com/cylc/cylc-admin/pull/40)
+[cylc-admin work plan](../proposal-rose-suite-run.md)
+
+## Purpose
+As part of the work to transfer functionality from `rose suite run` to cylc
+is is proposed that the options available in global, user and suite config
+files are brought into alignment. This file sets out a specification for the
+combined `cylc-flow.rc` file.
+
+## Preabamble
+
+To reduce changes required for end users this file format is based on
+`suite.rc`: Where aspects of the file are described as unchanged this implies
+that file contents are exactly the same as they would be in a `suite.rc`.
+Users of the `global.rc` are usually admins or power users and thus changes
+to the format of this file are preferable.
+
+It should be noted that although it will be possible to modify all settings in
+all contexts, that some settings are more likely to be used in global contexts
+and some are more likely to be used in suite configs.
+
+****
+
+## specification for `cylc-flow.rc`
+
+### Top level sections to copy from `suite.rc` to `cylc-flow.rc`
+
+Most sites will leave these to users (Although I could imagine adding a copy-
+right message to meta by default, for example, and one user has suggested a
+very simple runtime might be added too, for training and debugging purposes.)
+
+```ini
+[meta]
+  ...
+[scheduling]
+  ...
+[vizualization]
+  ...
+```
+
+### Top level sections to copy from `global.rc` to `cylc-flow.rc`
+
+It is likely that most users will continue to have these set by site admins.
+
+```ini
+[task events]
+  ...
+[test battery]
+  ...
+[suite host self-identification]
+  ...
+[authentication]
```

Comment:
Should become `[authorisation]`.

Reply:
Agreed, with spelling "authorization" which is acceptable (if archaic) in all forms of English.

Reply:
That's a truly beautiful depiction of a can of worms, @wxtim :laughing: Thank you for your commitment to humour!